packages/compass-e2e-tests/helpers/commands/wait-for-open-modal.ts

```diff
@@ -0,0 +1,34 @@
+import { inspect } from 'node:util';
+import type { WaitForOptions } from 'webdriverio';
+import type { CompassBrowser } from '../compass-browser';
+
+export async function waitForOpenModal(
+  browser: CompassBrowser,
+  selector: Parameters<CompassBrowser['$$']>[0],
+  { reverse = false, ...options }: WaitForOptions = {}
+): Promise<void> {
+  await browser.waitUntil(
+    async () => {
+      const modals = await browser.getOpenModals(selector);
+      if (reverse) {
+        return modals.length === 0;
+      } else {
+        for (const modal of modals) {
+          // Ensure any modals are interactable if open
+          await modal.waitForClickable({
+            timeout: 500,
```

Comment:
Unexpected to see defaults overriden, we configure them for the whole test suite via env usually because different environments might require slightly different configuration (we're also pretty generous with those usually especially for CI), so setting it to a value different from default by default doesn't seem right to me: it might be 500ms on your machine, but 5s next time windows host in GHA has issues with performance